root/artist/ArtistMerge.js

```diff
@@ -40,7 +40,7 @@ const buildMergeTarget = (artist, index) => (
   </li>
 );
 
-const MergeArtists = ({$c, form, toMerge}: Props) => (
+const ArtistMerge = ({$c, form, toMerge}: Props) => (
```

Comment:
Nit-picking: It should probably be `ArtistsMerge` as it requires at least two artists, plus it would be consistent with `ArtistsList`.

Reply:
I'd do all singular, but +1 for consistency. Plural noun adjuncts usually don't sound right in English (though there are exceptions, ArtistsList is not one of them and sounds awkward and ungrammatical, as does ArtistsMerge). I was gonna comment on this before too, but didn't since the others were already plural. :)

Reply:
I wouldn't mind changing them all to EntityList instead of EntitiesList, but maybe after we're done merging them? Then it can all be done with one commit.